integration/oasis/chargingstrategy/null_charge_strategy.go

```diff
@@ -0,0 +1,17 @@
+package chargingstrategy
+
+type nullChargeStrategy struct {
```

Comment:
same issue for naming. see https://github.com/golang/go/wiki/CodeReviewComments#package-names